swan-lake/integration-tools/openapi-tool.md

```diff
@@ -285,6 +287,71 @@ $ bal openapi -i <openapi-contract> --mode client --client-methods <resource|rem
+| `options.statusCodeBinding`| This option can be used in the client generation to generate the client methods with status code response binding.                                                                                                                                                                                                                                                                                                                                                                                           |  Optional         |
+
+
+#### Update the Ballerina.toml file with OpenAPI tool configurations using OpenAPI CLI command
+
+Executing the following OpenAPI `add` sub-command, along with the OpenAPI to Ballerina CLI options, will update the `Ballerina.toml` with OpenAPI tool configurations.
+
+```
+$ bal openapi add [-i | --input] <openapi-contract-file-path>
+                  [--id] <client-id>
+                  [--module] <target-module>
+                  [--package] <ballerina-package-path>
+```
+
+For example,
+
+```
+$ bal openapi add -i <openapi-contract> --id <client_id> --module <target-module> --package <ballerina package path> --tags <"tag1","tag2">
+```
+
+| Command option      | Description                                                                                                                                                                                                                                                                                                                                                                     | Mandatory/Optional |
+|----------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------|
+| `-i \| --input`   | The `-i|--input` command option specifies the path of the OpenAPI contract file (e.g., `my-api.yaml` or `my-api.json`).                                                                                                                                                                                                                                                                                                                                         | Mandatory          |
```

Comment:
Incorrect table line content it seems. 

<img width="1091" alt="image" src="https://github.com/ballerina-platform/ballerina-dev-website/assets/16300038/41b41761-cc61-4c05-b59d-d7fe1034f2fd">